pkg/ccl/backupccl/backupresolver/targets.go

```diff
@@ -306,7 +306,7 @@ func NewDescriptorResolver(descs []catalog.Descriptor) (*DescriptorResolver, err
 // session database) or if one of its tables matches the targets. All expanded
 // DBs, via either `foo.*` or `DATABASE foo` are noted, as are those explicitly
 // named as DBs (e.g. with `DATABASE foo`, not `foo.*`). These distinctions are
-// used e.g. by RESTORE.
+// used e.g. by RESTORE. tablePatternMap will be populated if non-nil.
```

Comment:
is it really worth it to optimize?   Perhaps just return this?

Reply:
+1

Reply:
Done, thanks.